### DISCUSSION TITLE
Has anyone ever hired TerriaJS for tech support?

### DISCUSSION THREAD

**Discussion by lstillwe** [TerriaJS/terriajs#6239] [General] [answered]
Hello! Has anyone in the community has ever hired TerriaJS folks for technical help in building a custom application?
I have tried contacting them via the email on the TerriaJS website, but have received no response.

Thanks very much.

**Comment by AnaBelgun** [comment, marked as answer]
hi @lstillwe thank you for reaching out. TerriaJS core team has been contracted in the past for building Terria based platforms. We are currently un-available but happy to continue the discussion on email. Unfortunately I cannot find your email/communication. Would you please resend it to info@terria.io. Thank you. Kind regards, Ana
